### PR TITLE
Add mod rules to allow smooth mesh parameters to be changed.

### PR DESCRIPTION
**Pull request by marcushutchings**
Add mod rules to allow smooth mesh to be configured.

### CODE REVIEW

File: rts/Sim/Misc/ModInfo.h [hunk 1]
```diff
@@ -220,6 +220,14 @@ class CModInfo
 
 	bool enableSmoothMesh;
 
+	/// Reduce the resolution of the smooth mesh by the divider value. Minimum 1, default 2.
```
Comment (sprunk) [inline rts/Sim/Misc/ModInfo.h:223]: The main reason to increase this would be performance, right?

Comment (marcushutchings) [reply rts/Sim/Misc/ModInfo.h]: Yup. Added comment.

**Review by sprunk — APPROVED** [review]

**Comment by marcushutchings** [comment]
Has the flattening happened or is it underway?

**Comment by sprunk** [comment]
It's underway, but putting them in the subtable first and moving them later means that there's one more thing to keep backwards compatibility for. It's not a big deal though.

**Pull request merged**